fdbserver/DataDistribution.actor.cpp

```diff
@@ -765,6 +765,7 @@ struct DDTeamCollection : ReferenceCounted<DDTeamCollection> {
 				} else {
 					self->medianAvailableSpace = SERVER_KNOBS->MIN_AVAILABLE_SPACE_RATIO;
 				}
+				TraceEvent("DDMedianAvailableSpace", self->distributorId).detail("Primary", self->medianAvailableSpace);
```

Comment:
It may be useful to include the number of teams being considered as well, as it may not be all of them.